components/create_post/create_post.jsx

```diff
@@ -56,43 +126,27 @@ export default class CreatePost extends React.Component {
         this.handleFileUploadComplete = this.handleFileUploadComplete.bind(this);
         this.handleUploadError = this.handleUploadError.bind(this);
         this.removePreview = this.removePreview.bind(this);
-        this.onChange = this.onChange.bind(this);
-        this.onPreferenceChange = this.onPreferenceChange.bind(this);
         this.getFileCount = this.getFileCount.bind(this);
         this.getFileUploadTarget = this.getFileUploadTarget.bind(this);
         this.getCreatePostControls = this.getCreatePostControls.bind(this);
         this.handleKeyDown = this.handleKeyDown.bind(this);
         this.handleBlur = this.handleBlur.bind(this);
         this.sendMessage = this.sendMessage.bind(this);
-        this.focusTextbox = this.focusTextbox.bind(this);
-        this.showPostDeletedModal = this.showPostDeletedModal.bind(this);
-        this.hidePostDeletedModal = this.hidePostDeletedModal.bind(this);
-        this.showShortcuts = this.showShortcuts.bind(this);
-        this.handleEmojiClick = this.handleEmojiClick.bind(this);
         this.handlePostError = this.handlePostError.bind(this);
```

Comment:
Better if these remaining bind functions are changed into its equivalent arrow functions.

Reply:
Yes of course.